src/monmove.cpp

```diff
@@ -277,10 +279,57 @@ float monster::rate_target( Creature &c, float best, bool smart ) const
     return INT_MAX;
 }
 
+bool monster::is_item_lootable( const item &itm )
+{
+    bool lootable = false;
+
+    //TODO: Definitely find a better way to do the below, too much repeatability
+
+    if( !lootable_categories.empty() )
+        for( std::string tag : lootable_categories ) {
+            if( itm.get_category().name() == tag ) {
```

Comment:
`name()` returns a translated string, so it's different when the player uses a different language. The check won't work anymore.